TsdrApi.php

```diff
@@ -50,16 +50,19 @@ public static function getApiForm()
 <h1>Trademark API test environment</h1>
 <p>Welcome to the Trademark API Test Environment!</p>
 <p>This form will dynamically retrieve relative information about any existing trademark on the fly!</p>
-<p>Please enter a serial number.</p>
-<i>If you don't have have one, use this for an example: 85129597</i>
+<p>Please enter a serial or registration number.</p>
+<i>If you don't have have one, use this for an example: serial no 85129597</i>
 <form method="post" action="$phpself">  
  <!-- Name: <input type="text" name="name">
   <br><br>
   E-mail: <input type="text" name="email">
   <br><br> -->
-  Serial Number: <input type="text" name="serial">
+Type: <input type="radio" name="type" value="sn" checked>Serial
```

Comment:
Let's use a **constant** here for "sn"

Reply:
I couldn't use the constant inside the heredoc without adding $constant in the change I pushed.